src/main/java/io/github/thebusybiscuit/slimefun4/utils/SlimefunUtils.java

```diff
@@ -288,6 +287,19 @@ public static boolean isItemSimilar(@Nullable ItemStack item, @Nullable ItemStac
 
                 ItemMetaSnapshot meta = ((SlimefunItemStack) sfitem).getItemMetaSnapshot();
                 return equalsItemMeta(itemMeta, meta, checkLore);
+            } else if (sfitem instanceof ItemStackWrapper && sfitem.hasItemMeta()) {
+                // Slimefun items may be ItemStackWrapper's in the context of cargo so let's try to do
+                // an ID comparison before meta comparison
```

Comment:
multiline comment :eyes:

```suggestion
                /*
                 * Slimefun items may be ItemStackWrapper's in the context of cargo so let's try to do
                 * an ID comparison before meta comparison
                 */
```